pppd: Support "nomp" option even if multilink support is off

This patch moves the "nomp" option entry outside of the defines protecting
the multilink specific code. The motivation is to allow "nomp" even if pppd
does not support multilink, so that controlling programs can unconditionally
pass it to pppd regardless of the compile time features.

Signed-off-by: Jo-Philipp Wich <jo@mein.io>

--- a/pppd/options.c
+++ b/pppd/options.c
@@ -336,13 +336,14 @@ option_t general_options[] = {
       "Enable multilink operation", OPT_PRIOSUB | OPT_ALIAS | 1 },
     { "nomultilink", o_bool, &multilink,
       "Disable multilink operation", OPT_PRIOSUB | 0 },
-    { "nomp", o_bool, &multilink,
-      "Disable multilink operation", OPT_PRIOSUB | OPT_ALIAS | 0 },
 
     { "bundle", o_string, &bundle_name,
       "Bundle name for multilink", OPT_PRIO },
 #endif /* HAVE_MULTILINK */
 
+    { "nomp", o_bool, &multilink,
+      "Disable multilink operation", OPT_PRIOSUB | OPT_ALIAS | 0 },
+
 #ifdef PLUGIN
     { "plugin", o_special, (void *)loadplugin,
       "Load a plug-in module into pppd", OPT_PRIV | OPT_A2LIST },
